lib/TokenManager.ts

```diff
@@ -166,8 +177,8 @@ function renew(sdk, tokenMgmtRef, storage, key) {
         // renewed token, we verify the promise key doesn't exist and return.
         return;
       }
-      add(sdk, tokenMgmtRef, storage, key, freshToken);
-      tokenMgmtRef.emitter.emit('renewed', key, freshToken, oldToken);
+      add(sdk, tokenMgmtRef, storage, key, freshToken, false /* shouldEmitEvent */);
```

Comment:
To follow up on above, I'd love to see this be:
`add({ sdk, tokenMgmtRef, storage, key, token: freshToken, shouldEmitEvent: false });`